bands/templates/bands/agreement-base.html

```diff
@@ -74,7 +74,9 @@
         <p>
             {% block band_text_accommodations_inner %}
             {{ band.num_hotel_rooms }} hotel room{{ band.num_hotel_rooms|pluralize }} with 2 queen beds
-            will be provided, with check in and out times provided as part of our final agreement.
+            will be provided, with check in on {{ c.EPOCH|datetime_local("%A, %B %-d") }}
+            and check out on {{ c.ESCHATON|datetime_local("%A, %B %-d") }}, unless
+            otherwise explicitly arranged via email.
```

Comment:
I set the hotel check in and out dates to EPOCH and ESCHATON, as those line up with the requested dates:

![hotels](https://user-images.githubusercontent.com/2592431/29694359-a31e0ff4-8909-11e7-9c8a-26ecb1d350df.PNG)